### PR TITLE
Renamed DEBUG to LOGGING in mtcp.

### PR DESCRIPTION
**Pull request by karya0**
This was leftover from cf53732.

### CODE REVIEW

File: src/mtcp/mtcp_util.h [hunk 1]
```diff
@@ -35,11 +35,11 @@
     mtcp_abort();                                       \
   }
 
-#ifdef DEBUG
+#ifdef LOGGING
```
Comment (rohgarg) [inline src/mtcp/mtcp_util.h]: Is the `LOGGING` macro available in mtcp? Do we need to change `Makefile.in` in the mtcp directory? Could you also see if we need to fix Makefile.debug in the mtcp directory?

Comment (karya0) [inline src/mtcp/mtcp_util.h]: Good catch! I'll take a look and update it.

Comment (karya0) [inline src/mtcp/mtcp_util.h]: It is available:

    gcc -I../../include -I. -DLOGGING -g -O2 -DLOGGING -c -o mtcp_restart.o\
      -DPIC -fPIC -fno-stack-protector -g -O0 mtcp_restart.c

I'll update Makefile.debug.

Comment (rohgarg) [inline src/mtcp/mtcp_util.h]: Great! Thank you!

Comment (karya0) [inline src/mtcp/mtcp_util.h]: Updated the PR. Should be good to go now.

**Pull request merged**